### ISSUE TITLE
Parallel on Windows not Working

### ISSUE THREAD

**Issue by DE0CH** [auto-optimization/iracepy#16]
CI works for macOS and ubuntu, but for some reason windows just does not work. It fails after 50 minutes with a strange memory error from pytest (not even the test themselves). 

I've removed the CIs on windows just so we can have a working CI before we can figure out what's wrong with it.

**Comment by MLopez-Ibanez** [comment]
Can you enable it but just for one python version? This way we get reminded that it needs to be fixed but also we can check whether it gets fixed.

**Comment by DE0CH** [comment]
Yep. But I'll probably put this (along with other comfort features) aside for a few weeks, since final exams are coming soon, and I will be probably short one time. Hope that's okay!

**Comment by DE0CH** [comment]
With commit f71e759 (and its issue #21), I figured out why it was failing on Windows. It fails with `parallel` > 1. I am not sure why but I don't want to investigate it now because I am planning to write a `targetRunnerParallel` in python that uses a worker pool anyways to improve performance so this should be fixed along with it too.

**Comment by MLopez-Ibanez** [comment]
Great you found out about this. I would be happy if you force `parallel=1` in windows with a warning telling people to use Linux or MacOS (or the R package) for parallel computation.

**Comment by DE0CH** [comment]
Thanks. I would prefer to just raise an `NotImplemented` error and ask the user to change it themselves, since this gets the user's attention and a warning can be hard to find in a sea of debugging information, leading to frustrations.

**Comment by DE0CH** [comment]
sorry, clicked the wrong close button.

**Comment by DE0CH** [comment]
@MLopez-Ibanez Seems like a fix is not very straightforwad and #22 has its own problems. Should we close this as not planned?
